lib/linter-rust.coffee

```diff
@@ -273,19 +140,32 @@ class LinterRust
           result.push ['--cfg', "feature=\"#{f}\""]
         result
 
-  ableToJSONErrors: (curDir) =>
+  decideErrorMode: (curDir, commandMode) =>
     # current dir is set to handle overrides
-
     sb_exec.exec(@rustcPath, ['--version'], {stream: 'stdout', cwd: curDir, stdio: 'pipe'}).then (stdout) =>
```

Comment:
The same can be said for other "check calls", such as [here](https://github.com/AtomLinter/linter-rust/pull/84/files#diff-b6440b005dd344507743aaaa07d74383R168)